sgkit/model.py

```diff
@@ -15,43 +16,46 @@
 def create_genotype_call_dataset(
     *,
     variant_contig_names: List[str],
-    variant_contig: Any,
-    variant_position: Any,
-    variant_alleles: Any,
-    sample_id: Any,
-    call_genotype: Any,
-    call_genotype_phased: Any = None,
-    variant_id: Any = None,
+    variant_contig: ArrayLike,
```

Comment:
I don't think this is under our control, but the way it displays in the HTML is as `ndarray, Array`, then when you mouse over it changes to `Union[ndarray, Array]`.

Reply:
Yes, indeed. `sphinx_autodoc_typehints` does that to make it more readable, as `ndarray, Array` is easier on eyes than `Union[ndarray, Array]`

Reply:
and moreover, just displaying `ArrayLike` doesn't tells the user explicitly what it means.